packages/cli-lib/lib/constants.js

```diff
@@ -118,6 +118,49 @@ const PROJECT_TEMPLATE_TYPES = {
   },
 };
 
+const TEMPLATE_TYPES = {
```

Comment:
Not necessarily a blocker, but it would be nice if this was something that the BE could return to us so that we could keep a single source of truth. I don't imagine that it will change frequently, but it'll be yet another area where we have to manually keep data in sync.